libcodechecker/cmd/cmd_line_client.py

```diff
@@ -104,9 +129,13 @@ def add_filter_conditions(report_filter, filter_str):
         map(lambda x: x.strip(), filter_str.split(':'))
 
     if severities:
+        if not valid_filter_value(severities,
+                                  ttypes.Severity._NAMES_TO_VALUES):
+            sys.exit(1)
```

Comment:
This way if a filter contain more than one invalid filter than the first non valid filter values will be written only.
```bash
CodeChecker cmd results --filter "bad_sev:::bad_dt:bad_rw" test
```
In this example only invalid severity values are written to the console.
It would be great to write out all invalid values: `bad_dt` and `bad_rw` too.
E.g.:
```py
    if not all(valid for valid in [valid_filter_value(x[0], x[1]) for x in [
        (severities, ttypes.Severity._NAMES_TO_VALUES),
        (dt_statuses, ttypes.DetectionStatus._NAMES_TO_VALUES),
        (rw_statuses, ttypes.ReviewStatus._NAMES_TO_VALUES)]]):
        sys.exit(1)
```